cypress/support/utils/cypressTools.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
+export function getLongDelay() {
```

Comment:
May be better to create class instead of funtion and describe all our timeouts like default, long or may be others

Reply:
I can't see benefits from class wrapping. Could you describe the reason?
" describe all our timeouts like default, long or may be others" - can't understand you, give an example please

Reply:
Now we save our timeouts in different places,
For example one timeout in cypress.json another timeout in cypressTools
I suggest to use one place for timeouts
May be this is not a class, may be use another way for it

Reply:
additionally we have timeout related with interactors. For me - it's ok for now and may be turned in the future(for example as set of env variables)

Reply:
I've found that we use in our code different places with different long timeouts(you can see it into original code in this PR). And my change related with wrapping long timeout into one place(function) and continue to use it without doubts which value should be specified if it's needed.

Reply:
ok, I agree with you to think about it in future

Reply:
thank's